pkg/provider/cloud/openstack/provider.go

```diff
@@ -123,7 +116,7 @@ func (os *Provider) DefaultCloudSpec(ctx context.Context, spec *kubermaticv1.Clu
 }
 
 func (os *Provider) ClusterNeedsReconciling(cluster *kubermaticv1.Cluster) bool {
-	return false
+	return true
```

Comment:
This cannot be set to true like this; you can have it `true` in a conditional manner though similar to https://github.com/kubermatic/kubermatic/blob/main/pkg/provider/cloud/aws/provider.go#L108-L116

But do we really need this change?